examples/count-linear-extensions-using-hpolytope/instances/bipartite_0.5_008_0.txt

```diff
@@ -0,0 +1,8 @@
+0 0 0 0 1 0 1 0
```

Comment:
could you add the source of the file and how you generated in a comment?

Reply:
I generated it using the instances given in the [le-counting-practice](https://github.com/vaithak/le-counting-practice) repo. Should I add the details in README file or a comment in C++ source code?

Reply:
ALso I was thinking of adding the plots of corresponding DAGs of these instances in the README file.

Reply:
>I generated it using the instances given in the le-counting-practice repo. Should I add the details in README file or a comment in C++ source code?

README should be fine